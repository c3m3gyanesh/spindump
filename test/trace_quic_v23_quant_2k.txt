2k file transfer using Quant, with version draft-23. Some retransmissions happened, probably.
