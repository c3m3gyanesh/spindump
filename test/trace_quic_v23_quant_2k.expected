QUIC 172.16.95.160 <-> 52.58.13.57 04b8e737-7f9b3c260af1ac9b4d99c54e864fde6bdb (57237:4433) at 1571074054421142 new starting packets 0 0 bytes 0 0
QUIC 172.16.95.160 <-> 52.58.13.57 04b8e737-0872420297a67c03 (57237:4433) at 1571074061613179 measurement up right 187906 packets 4 0 bytes 4912 0 bandwidth 1228 0
QUIC 172.16.95.160 <-> 52.58.13.57 7e86ccba-0872420297a67c03 (57237:4433) at 1571074061988966 measurement up right 184905 packets 8 13 bytes 5283 13573 bandwidth 1228 13573
QUIC 172.16.95.160 <-> 52.58.13.57 7e86ccba-0872420297a67c03 (57237:4433) at 1571074061993350 measurement up left 4384 packets 8 21 bytes 5283 22504 bandwidth 1228 22504
QUIC 172.16.95.160 <-> 52.58.13.57 7e86ccba-0872420297a67c03 (57237:4433) at 1571074061993350 measurement up full (initiator) 189289 packets 8 21 bytes 5283 22504 bandwidth 1228 22504
QUIC 172.16.95.160 <-> 52.58.13.57 7e86ccba-0872420297a67c03 (57237:4433) at 1571074062181652 measurement up right 188302 packets 11 21 bytes 5458 22504 bandwidth 1228 22504
QUIC 172.16.95.160 <-> 52.58.13.57 7e86ccba-0872420297a67c03 (57237:4433) at 1571074062181652 measurement up full (responder) 192686 packets 11 21 bytes 5458 22504 bandwidth 1228 22504
